world/dominion/plots/models.py

```diff
@@ -1064,6 +1064,14 @@ class PlotAction(AbstractAction):
         related_name="actions",
         on_delete=models.SET_NULL,
     )
+    episode = models.ForeignKey(
+        "character.Episode",
+        db_index=True,
+        blank=True,
+        null=True,
+        related_name="episode",
```

Comment:
related_name is referring to the objects that point to the episode, so episode is misleading. It should also be plural, since a FK reflects a many-to-one relationship, with the episode being the "one" side of that. Since this is a plot action, it should probably be `"actions"` or `"plot_actions"`.

Reply:
This still needs fixed